docs/getting-started/sketch.md

```diff
@@ -46,9 +46,14 @@ Mark any visuals you want to use in your Diez project for export (or create slic
 ### Syncing your Sketch file with your Diez project
 Make sure to save your changes. Now you're all set up to sync your Sketch file with your Diez project. Head back over to your terminal and within your Diez project folder run:
 
-```bash
-yarn diez extract
+<CodeTabs>
+```bash tabname-yarn
+$ yarn diez extract
 ```
+```bash tabname-npm
+$ npx diez extract
```

Comment:
@roperzh Just curious why this command is npx & not npm?

Reply:
In this particular case, `npx` is they npm-way to run a locally installed binary (in this case the repo copy of `diez`), for this purposes it is the equivalent of running `yarn diez ...`.

However, if you want to run a script in your `package.json` for example, `npm run script-name` is the way to go.